hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/contract/precompile/CreatePrecompileSuite.java

```diff
@@ -115,16 +144,570 @@ List<HapiSpec> positiveSpecs() {
+                                .exposingKeyTo(k -> ed2551Key.set(k.getThresholdKey()
+                                        .getKeys()
+                                        .getKeys(0)
+                                        .getEd25519()
+                                        .toByteArray())),
+                        cryptoUpdate(ACCOUNT).key(THRESHOLD_KEY))
+                .when(withOpContext((spec, opLog) -> allRunFor(
+                        spec,
+                        getContractInfo(TOKEN_CREATE_CONTRACT)
+                                .has(ContractInfoAsserts.contractWith().autoRenewAccountId(ACCOUNT))
+                                .logged())))
+                .then(withOpContext((spec, ignore) -> {
+                    final var subop1 = balanceSnapshot(ACCOUNT_BALANCE, ACCOUNT);
+                    final var subop2 = contractCall(
+                                    TOKEN_CREATE_CONTRACT,
+                                    CREATE_NFT_WITH_KEYS_AND_EXPIRY_FUNCTION,
+                                    HapiParserUtil.asHeadlongAddress(
+                                            asAddress(spec.registry().getAccountID(ACCOUNT))),
+                                    ed2551Key.get(),
+                                    HapiParserUtil.asHeadlongAddress(
+                                            asAddress(spec.registry().getAccountID(ACCOUNT))),
+                                    AUTO_RENEW_PERIOD)
+                            .via(FIRST_CREATE_TXN)
+                            .gas(GAS_TO_OFFER)
+                            .payingWith(ACCOUNT)
+                            .sending(DEFAULT_AMOUNT_TO_SEND)
+                            .refusingEthConversion()
+                            .exposingResultTo(result -> {
+                                log.info("Explicit create result is" + " {}", result[0]);
+                                final var res = (Address) result[0];
+                                createdNftTokenNum.set(res.value().longValueExact());
+                            })
+                            .hasKnownStatus(SUCCESS);
+
+                    allRunFor(
+                            spec,
+                            subop1,
+                            subop2,
+                            childRecordsCheck(
+                                    FIRST_CREATE_TXN,
+                                    SUCCESS,
+                                    TransactionRecordAsserts.recordWith().status(SUCCESS)));
+
+                    final var nftInfo = getTokenInfo(asTokenString(TokenID.newBuilder()
+                                    .setTokenNum(createdNftTokenNum.get())
+                                    .build()))
+                            .hasAutoRenewAccount(ACCOUNT)
+                            .logged();
+
+                    allRunFor(spec, nftInfo);
+                }));
+    }
+
+    // TEST-001
```

Comment:
Is this the same TEST number as above?

Reply:
I think so, because both of the tests call the same contract function named `createTokenWithKeysAndExpiry` and marked as ` TEST-001`